### PR TITLE
Move crons from Symfony Cloud to Github Actions

### PR DESCRIPTION
**Pull request by jordisala1991**
Closes https://github.com/sonata-project/dev-kit/issues/820

Symfony Cloud will still be needed by only because of the webhook, and that does not need frequent deploys, because it only does a comment on pull request that need a rebase because have conflicts

Someone (@greg0ire or @OskarStark) will need to create the secrets on this repository settings

### CODE REVIEW

File: .github/workflows/dispatch-with-files.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Dispatch with files
+
+on:
+  schedule:
+    - cron: '5 2 * * *'
+
+  workflow_dispatch:
```
Comment (jordisala1991) [inline .github/workflows/dispatch-with-files.yaml:7]: In addition to cron, we will have a button on the github actions interface to trigger manually the dispatch with files (and the normal dispatch)

File: .github/workflows/auto-merge.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          key: ${{ runner.os }}-composer
+
+      - name: Install Composer dependencies
+        run: composer install --prefer-dist --no-progress --no-interaction
+
+      - name: Dispatch files
+        run: bin/console auto-merge --apply
+        env:
+          GITHUB_OAUTH_TOKEN: ${{ secrets.GITHUB_OAUTH_TOKEN }}
+          DEK_KIT_TOKEN: ${{ secrets.DEK_KIT_TOKEN }}
```
Comment (greg0ire) [inline .github/workflows/auto-merge.yaml:44]: Maybe we should take this as an occasion to fix this typo:
```suggestion
          DEV_KIT_TOKEN: ${{ secrets.DEV_KIT_TOKEN }}
```

or at least not propagate it further?

```suggestion
          DEK_KIT_TOKEN: ${{ secrets.DEV_KIT_TOKEN }}
```

**Review by OskarStark — APPROVED** [review]

File: .github/workflows/auto-merge.yaml (hunk 2 above)
Comment (jordisala1991) [reply .github/workflows/auto-merge.yaml]: Done. I choose the second option. I prefer to fix that mess on separate PR, let's not propagate further tho.

**Comment by jordisala1991** [comment]
@OskarStark with all the changes to dev-kit you made, is this something you want to move forward or you prefer to implement auto deploy to symfony cloud?

**Comment by OskarStark** [comment]
I prefer auto deploy to symfony cloud to be honest :-*

**Comment by jordisala1991** [comment]
Lets close then

**Comment by VincentLanglet** [comment]
What is needed for auto deploy then ?

**Pull request closed without merging**